src/common/notifications/NotificationsProvider.js

```diff
@@ -0,0 +1,36 @@
+import React, { useRef, useState } from 'react';
+import Notifications from './Notifications';
+
+export const NotificationsContext = React.createContext({});
+
+/**
+ * Viser grønn/rød alert-stripe på toppen av siden, når
+ * andre komponenter bruker notify('din melding');
+ */
```

Comment:
Denne NotificationsProvider er ikke koblet til redux. Når andre komponenter etterhvert kobles fra redux, så kan de også bruke denne for å kjøre notify.